src/main/java/eu/bbmri_eric/negotiator/api/controller/v3/tasks/ScheduledDirectorySyncTask.java

```diff
@@ -0,0 +1,19 @@
+package eu.bbmri_eric.negotiator.api.controller.v3.tasks;
+
+import eu.bbmri_eric.negotiator.service.MolgenisDirectorySyncService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ScheduledDirectorySyncTask {
```

Comment:
move this to a different package /negotiator/scheduled and rename it to DirectorySynchronization.